test/src/test/java/test/compatability/BridgeMethodsTest.java

```diff
@@ -0,0 +1,36 @@
+package test.compatability;
```

Comment:
s/compatability/compatibility

Reply:
fixed in b36eec37598edfd20dc063e5380421befe1c03f0